Runner/suites/Kernel/Baseport/BWMON/BWMON.yaml

```diff
@@ -0,0 +1,21 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: AUSanity
+    description: "Baseline AU Sanity test plan"
```

Comment:
Please update the name and description to correspond with what the test script does.